src/renderer/composables/lang.ts

```diff
@@ -0,0 +1,14 @@
+import { ref } from 'vue'
+import { Lang, TRANSLATIONS } from '../consts'
+import _ from 'lodash'
+
+const lang = ref<Lang>("sk")
+
+export const trans = (path: string) => {
+    return _.get(TRANSLATIONS[lang.value], path)
+}
+
+export function useLang() {
+    const toggle = () => { lang.value === "sk" ? lang.value = "en" : lang.value = "sk" }
```

Comment:
I find setting more tractable than toggling. There is nothing inherently 'toggleable' about switching language (perhaps just the UI might suggest it)
```suggestion
    const setLang = (lang: 'sk' | 'en' ) => { lang.value === lang }
```